src/pages/HomePage.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+//fetching constants and functions
+import VEGETABLE_ITEMS from '../data/vegetablesData.json';
+import {FILTER_TYPES, CART_STATUS} from '../data/constData';
+import { incrementItemInCartHandler, decrementItemInCartHandler } from './components/atoms/itemCartBtnHelperFunctions';
+import { searchVegetable } from './components/organism/header/molecules/atoms/SearchBar';
+import { filterHandler } from './components/homePage/mainSection/molecules/atoms/Filter';
+
+//importing components
+import Navbar from './components/organism/header/Navbar';
+import SubHeaderCategories from './components/homePage/molecules/subHeaderCategories/SubHeaderCategories'
```

Comment:
We should have an index file for every component folder, so that we have single source for exposing our data out of that folder